src/commands/misc/helpRemember.js

```diff
@@ -1,44 +1,56 @@
-const { EmbedBuilder } = require('@discordjs/builders');
+const { SlashCommandBuilder, EmbedBuilder } = require('@discordjs/builders');
```

Comment:
Doesn't have to be done in this PR, but it would be helpful to see if these can be added dynamically